tests/Doctrine/Tests/DBAL/Functional/Schema/PostgreSqlSchemaManagerTest.php

```diff
@@ -514,6 +514,17 @@ public function testAlterTableAutoIncrementIntToBigInt(string $from, string $to,
         self::assertTrue($tableFinal->getColumn('id')->getAutoincrement());
     }
 
+    public function testCommentInTable() : void
```

Comment:
It doesn't look like a Postgres-specific feature. Would it be possible to move this test in the base class and make sure it works the same across platforms?

Reply:
It would really be great to have this work across all platforms the same way. If it did by the way, we could probably create a "getComments" method in the `Table` object instead of resorting to options.
I won't have to much time to work on this in the coming weeks however. Maybe we could merge this first and I'll open another PR for other platforms if I find some time? (or any help is welcome :) )

Reply:
Note: I opened a separate PR for the other databases here: https://github.com/doctrine/dbal/pull/3512
There is already a working support for Sqlite.

Reply:
We can merge it implemented for one platform but the functional test still should be run against all supported platforms. It can be marked incomplete for the ones which don't implement this feature.

This way, it will be clear that it's a cross-platform feature and the incomplete tests will indicate that there's some work to be done. Otherwise, we have tests like this scattered across the codebase and no clear picture of what is supported by the abstraction.

Reply:
Excellent idea.
I moved the tests in `SchemaManagerFunctionalTestCase` in https://github.com/doctrine/dbal/pull/3499/commits/4e8cefdcd163a66dabe90e66b39657fb41e33775 and marked the tests "incomplete" for the platforms not supported in  https://github.com/doctrine/dbal/pull/3499/commits/5eb857757e98a0242252187b023f53521e6bc0bc

Unless you have any further comments, I think this PR is ready.